src/styles/_main.scss

```diff
@@ -0,0 +1,28 @@
+@use './variables';
+
+.catalog {
+  --columns: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--columns), 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+  font-family: variables.$font-family-header;
```

Comment:
The font-family variable `variables.$font-family-header` is correctly used here for the catalog class, ensuring consistency with the header styling.